src/index.js

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client';
+import ReactDOM from 'react-dom';
 import './index.css';
-import { BrowserRouter } from 'react-router-dom';
-import reportWebVitals from './reportWebVitals';
-import App from './Components/App';
+import { Provider } from 'react-redux';
+import BookStore from 'components/BookStore';
+import store from './redux/configureStore';
```

Comment:
- [ ] There is a bug here with the Bookstore path, the app is starting with some errors. You would fix it by using `import BookStore from './Components/BookStore';`

<img width="1438" alt="Screenshot 2022-12-03 at 10 15 45 AM" src="https://user-images.githubusercontent.com/40503704/205433702-86d62e22-6e26-45fc-949a-591a3e0e0fa0.png">